src/org/jitsi/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimation.java

```diff
@@ -488,4 +566,144 @@ private synchronized void fireBandwidthEstimationChanged(
             second = b;
         }
     }
+
+    public class StatisticsImpl implements Statistics
```

Comment:
Does this come from webrtc.org?

Reply:
No

Reply:
In this case, can you add documentation, please?